src/AnnotationDialog.js

```diff
@@ -330,13 +331,45 @@ class AnnotationDialog extends EventEmitter {
         this.element.addEventListener('mouseup', this.stopPropagation);
         this.element.addEventListener('wheel', this.stopPropagation);
 
+        const replyTextEl = this.element.querySelector(`.${CLASS_REPLY_TEXTAREA}`);
+        if (replyTextEl) {
+            replyTextEl.addEventListener('keydown', this.focusReplyEl);
```

Comment:
Should we be listening to focus so that this also fires when you click in the input?

Reply:
I think it makes more sense to remove the red border once you type something into the text area rather than when you click into it

Reply:
Played around w/ it a little more. Changing it back to fire on 'focus' rather than 'keydown'